app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
+require 'lighthouse/veterans_health/client'
+
+module FastTrack
+  class DisabilityCompensationJob
+    include Sidekiq::Worker
+
+    extend SentryLogging
+    # TODO: This is apparently at most about 4.5 hours.
+    # https://github.com/mperham/sidekiq/issues/2168#issuecomment-72079636
+    sidekiq_options retry: 8
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      submission_id = msg['args'].first
+      submission = Form526Submission.new
+      submission.start_evss_submission(_status, submission_id: submission_id)
```

Comment:
How long does this method call typically take?

Reply:
As far as we've tested, it takes a few seconds (maybe 2 or 3), between the different Lighthouse API requests and the upload to S3.